behind-the-veil-siteroot/imports/ui/components/pages/service/AddEditServicePage.jsx

```diff
@@ -108,126 +140,253 @@ export const AddEditServicePage = ({ isEdit }) => {
+                        />
+                        {errors.serviceDescription &&
+                            <span className="text-cancelled-colour">{errors.serviceDescription}</span>}
+                    </div>
+
+                    {/* file upload area */}
+                    <div className="flex flex-col gap-1">
+                        <div className="flex flex-row">
+                            <Input
+                                className="text-opacity-0 border-r-0 rounded-r-none w-[130px]"
+                                type="file"
+                                accept={allowedFileTypeExtensions?.join(",")}
+                                label={<label className="main-text">Photos</label>}
+                                onChange={handleFileChange}
+                                onClick={(event) => {
+                                    // this is to allow the same file to be selected again after reset
+                                    event.target.value = null
+                                }}
+                                multiple
+                            />
+                            <TrashIcon
+                                className="input-base border-l-0 rounded-l-none h-[48px] w-[48px] mt-[28px] size-5 hover:bg-white-hover active:bg-light-grey-hover transition duration-200 ease-in-out"
+                                onClick={removeAllFiles}
+                            ></TrashIcon>
+                        </div>
+                        {fileRejected && <span className="text-cancelled-colour">{fileRejectedMessage}</span>}
+                        {errors.images && <span className="text-cancelled-colour">{errors.images}</span>}
                     </div>
-                    <div className="grid grid-cols-2 gap-2" id="preview">
+
+
+                    <div className="grid md:grid-cols-2 gap-5" id="preview">
                         {images}
                     </div>
 
                     <Button
                         className="flex bg-secondary-purple hover:bg-secondary-purple-hover mt-[15px]"
-                        onClick={AddEdit}
+                        onClick={handleSubmit}
                     >
-                        <CheckIcon className="size-[25px]" />
+                        <CheckIcon className="icon-base"/>
                         {button}
                     </Button>
                 </div>
             </div>
+
+            <Modal classNames={{
+                modal: "w-[550px] h-[300px] rounded-[45px] bg-glass-panel-background border border-main-blue"
+            }} open={open} onClose={onCloseModal} center showCloseIcon={false}>
+                <div className="flex flex-col justify-center items-center h-full gap-y-10">
+                    <h2 className="text-center title-text px-4">
+                        {(isEdit ? "Modification" : "Creation") + (isSuccess ? "was Successful" : " Failed")}
```

Comment:
just did a quick test, i think there's a missing space i.e. should be " was Successful"
![image](https://github.com/user-attachments/assets/de3be57e-9ba2-42c8-bf2d-dc437f28041c)